### PR TITLE
Document APF debug URL paths

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
Documented the debug paths introduced by https://github.com/kubernetes/kubernetes/pull/90967

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
/cc @yue9944882 
@lavalamp 
@deads2k 
@rhngy

**Comment by lavalamp** [comment]
Tech LGTM

/lgtm

File: content/en/docs/concepts/cluster-administration/flow-control.md [hunk 1]
```diff
@@ -365,9 +368,43 @@ poorly-behaved workloads that may be harming system health.
...
-
-
-
+### Debug endpoints
+
+When you enable the API Priority and Fairness feature, the kube-apiserver serves the following additional paths at its HTTP[S] ports.
+
+- `/debug/api_priority_and_fairness/dump_priority_levels` - a listing of all the priority levels and the current state of each.  Following is an example.
+  ```
+  $ kubectl get --raw /debug/api_priority_and_fairness/dump_priority_levels
```
Comment (sftim) [inline content/en/docs/concepts/cluster-administration/flow-control.md:377]: nit: it's better to [separate commands from their output](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output), and that also lets you turn on highlighting for the shell part.

**Comment by sftim** [comment]
/lgtm
echoing https://github.com/kubernetes/website/pull/22651#issuecomment-662718226

**Comment by sftim** [comment]
Looks like the static site generator OOM-ed.
It's a big site. I've triggered another build.

**Comment by MikeSpreitzer** [comment]
@MikeSpreitzer

File: content/en/docs/concepts/cluster-administration/flow-control.md [hunk 2]
```diff
@@ -365,9 +368,51 @@ poorly-behaved workloads that may be harming system health.
...
+  PriorityLevelName, Index,  PendingRequests, ExecutingRequests, VirtualStart,
+  workload-high,     0,      0,               0,                 0.0000,
+  workload-high,     1,      0,               0,                 0.0000,
+  workload-high,     2,      0,               0,                 0.0000,
+  ...
+  leader-election,   14,     0,               0,                 0.0000,
+  leader-election,   15,     0,               0,                 0.0000,
+  ```
+
+- `/debug/api_priority_and_fairness/dump_requests` - a listing of all the requests currently in the server, waiting or executing.  You can fetch like this:
```
Comment (yue9944882) [inline content/en/docs/concepts/cluster-administration/flow-control.md:406]: `includeRequestDetails` will dump the requests in an even finer granularity

File: content/en/docs/concepts/cluster-administration/flow-control.md [hunk 3]
```diff
@@ -303,6 +303,9 @@ to get a mapping of UIDs to names for both FlowSchemas and
 PriorityLevelConfigurations.
 
 ## Observability
+
+### Metrics
```
Comment (yue9944882) [inline content/en/docs/concepts/cluster-administration/flow-control.md:307]: need we revise the metrics section to reflect the recent metrics enhancements?

Comment (MikeSpreitzer) [reply content/en/docs/concepts/cluster-administration/flow-control.md:307]: Yes.  https://github.com/kubernetes/website/pull/22655

File: content/en/docs/concepts/cluster-administration/flow-control.md (hunk 2 above)
Comment (MikeSpreitzer) [reply content/en/docs/concepts/cluster-administration/flow-control.md:406]: Ah, right.  Revising...

**Comment by MikeSpreitzer** [comment]
The force-push to 0794fe200 add documentation of `/debug/api_priority_and_fairness/dump_requests?includeRequestDetails=1` and squashes.

File: content/en/docs/concepts/cluster-administration/flow-control.md (hunk 2 above)
Comment (MikeSpreitzer) [reply content/en/docs/concepts/cluster-administration/flow-control.md:406]: revised

**Comment by MikeSpreitzer** [comment]
The force-push to dc432198e corrects the characterization of `/debug/api_priority_and_fairness/dump_requests` and squashes.

**Pull request merged**